apps/meteor/app/livechat/server/lib/QueueManager.ts

```diff
@@ -62,33 +66,48 @@ export const queueInquiry = async (inquiry: ILivechatInquiryRecord, defaultAgent
 
 	if (dbInquiry.status === 'ready') {
 		logger.debug(`Inquiry with id ${inquiry._id} is ready. Delegating to agent ${inquiryAgent?.username}`);
+		/**
+		 * There is no need to call QueueManager.dispatchInquiryPosition
+		 * because the method is called on afterTakeInquiry anyway
+		 *
+		 */
 		return RoutingManager.delegateInquiry(dbInquiry, inquiryAgent, undefined, room);
 	}
+
+	await QueueManager.dispatchInquiryPosition(dbInquiry);
 };
 
-interface IQueueManager {
-	requestRoom: <
-		E extends Record<string, unknown> & {
-			sla?: string;
-			customFields?: Record<string, unknown>;
-			source?: OmnichannelSourceType;
-		},
-	>(params: {
-		guest: ILivechatVisitor;
-		rid?: string;
-		message?: string;
-		roomInfo: {
-			source?: IOmnichannelRoom['source'];
-			[key: string]: unknown;
-		};
-		agent?: SelectedAgent;
-		extraData?: E;
-	}) => Promise<IOmnichannelRoom>;
-	unarchiveRoom: (archivedRoom?: IOmnichannelRoom) => Promise<IOmnichannelRoom>;
-}
+export const QueueManager = class {
+	/**
+	 * It should be used when a inquiry is moved to queue(created|enabled|etc)
+	 * mainly used to inform visitors when they value is set for the first time
+	 *
+	 * @param inquiry
+	 *
+	 */
+	static async dispatchInquiryPosition(inquiry: ILivechatInquiryRecord) {
```

Comment:
This was "ee" code, I don't mind moving it but just making sure the change is intentional (ee => ce)